### PR TITLE
(FACT-2132) Implement disk facts for fedora. 

### PR DESCRIPTION
**Pull request by gigicampean**

### CODE REVIEW

**Comment by gigicampean** [comment]
Tests passed :)

File: spec/facter/facts/fedora/disk/sda/size_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+describe 'Fedora DiskSdaSize' do
+  context '#call_the_resolver' do
+    it 'returns a fact' do
+      expected_fact = double(Facter::ResolvedFact, name: 'disk.sda.size', value: '1.0 KiB')
+      allow(Facter::Resolvers::Linux::Disk).to receive(:resolve).with(:sda_size).and_return(1024)
+      allow(Facter::ResolvedFact).to receive(:new).with('disk.sda.size', '1.0 KiB').and_return(expected_fact)
+
+      fact = Facter::Fedora::DiskSdaSize.new
+      expect(Facter::BytesToHumanReadable.convert(1024)).to eq('1.0 KiB')
```
Comment (IrimieBogdan) [inline spec/facter/facts/fedora/disk/sda/size_spec.rb]: This line seams to test `BytesToHumanReadable ` not  fedora `disk.sda.size`

File: spec/facter/facts/fedora/disk/sr0/size_spec.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+describe 'Fedora DiskSr0Size' do
+  context '#call_the_resolver' do
+    it 'returns a fact' do
+      expected_fact = double(Facter::ResolvedFact, name: 'disk.sr0.size', value: '1.0 KiB')
+      allow(Facter::Resolvers::Linux::Disk).to receive(:resolve).with(:sr0_size).and_return(1024)
+      allow(Facter::ResolvedFact).to receive(:new).with('disk.sr0.size', '1.0 KiB').and_return(expected_fact)
+
+      fact = Facter::Fedora::DiskSr0Size.new
+      expect(Facter::BytesToHumanReadable.convert(1024)).to eq('1.0 KiB')
```
Comment (IrimieBogdan) [inline spec/facter/facts/fedora/disk/sr0/size_spec.rb]: This line seams to test `BytesToHumanReadable ` not  fedora `disk.sda.size`

Comment (IrimieBogdan) [reply spec/facter/facts/fedora/disk/sr0/size_spec.rb]: Maybe we can use a mock e.g.
`expect(Facter::BytesToHumanReadable).to receive(:convert).with(1024).and_return('1.0 KiB')`

**Review by BogdanIrimie — DISMISSED** [review]

**Review by BogdanIrimie — APPROVED** [review]

**Pull request merged**